build/helpers/ifEqual.js

```diff
@@ -0,0 +1,10 @@
+/* https://github.com/zeke/handlebars-helper-equal/blob/master/index.js */
+module.exports = function(lvalue, rvalue, options) {
+  if (arguments.length < 3)
```

Comment:
In this model, are we going to end up in a cycle of one-file-per-helper that then necessitates updates to the gulp task? I'd like to look and see if we can find another model because that seems onerous.